src/test/java/core/basesyntax/FruitShopTest.java

```diff
@@ -0,0 +1,248 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.createreport.CreateReport;
+import core.basesyntax.gettingreport.GetFromReportImpl;
+import core.basesyntax.gettingreport.GettingNewReportImpl;
+import core.basesyntax.service.StorageServiceImpl;
+import core.basesyntax.stoage.Storage;
+import core.basesyntax.tranzactions.BalanceTransaction;
+import core.basesyntax.tranzactions.DefineTheTransaction;
+import core.basesyntax.tranzactions.MakingDailyTransactionsImpl;
+import core.basesyntax.tranzactions.PurchaseTransaction;
+import core.basesyntax.tranzactions.ReturnTransaction;
+import core.basesyntax.tranzactions.SupplyTransaction;
+import java.util.List;
+import java.util.Map;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class FruitShopTest {
+    private MakingDailyTransactionsImpl transactions;
+    private GettingNewReportImpl toReport;
+    private GetFromReportImpl getFromReport;
+    private DefineTheTransaction definition;
+    private BalanceTransaction balance;
+    private PurchaseTransaction purchase;
+    private SupplyTransaction supply;
+    private ReturnTransaction returning;
+    private StorageServiceImpl service;
+    private CreateReport creating;
+
+    @BeforeMethod
+    public void setUp() {
+        service = new StorageServiceImpl();
+        transactions = new MakingDailyTransactionsImpl();
+        toReport = new GettingNewReportImpl();
+        getFromReport = new GetFromReportImpl();
+        definition = new DefineTheTransaction();
+        balance = new BalanceTransaction();
+        purchase = new PurchaseTransaction();
+        supply = new SupplyTransaction();
+        returning = new ReturnTransaction();
+        creating = new CreateReport();
+        Storage storage = new Storage();
+    }
+
+    @Test
+    public void transferringNonExistentFileName_NotOk() {
+        assertThrows(RuntimeException.class,() -> getFromReport.getFromReport("Non-Existent.csv"),
+                "File is not existent");
+    } //good
```

Comment:
Цих коментарів не повинно бути
```suggestion
    }
```